scripts/lib/browser-sdk-version.js

```diff
@@ -0,0 +1,9 @@
+const lernaJson = require('../../lerna.json')
+
+function getBrowserSdkVersion() {
+  return lernaJson.version
+}
+
+module.exports = {
+  getBrowserSdkVersion,
+}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏what about exporting the version directly instead of a getter?